client/webserver/locales/en-us.go

```diff
@@ -262,4 +262,12 @@ var EnUS = map[string]string{
 	"No Recent Activity":          "No Recent Activity",
 	"Recent asset_name Activity":  "Recent <span data-asset-name=1></span> Activity",
 	"Other Actions":               "Other Actions",
+	"estimated_fee":               "Estimated Fee",
+	"estimated_total_spend":       "Estimated Total Spend",
+	"estimated_balance":           "Estimated Balance After",
+	"max_estimated_send":          "Max Estimated Send",
+	"max_estimated_send_fee":      "Max Estimated Send Fee",
+	"sending":                     "Sending",
+	"transfer":                    "Transfer",
+	"invalid_address":             "Note: You have to provide a valid address before proceeding.",
```

Comment:
<s>Note: You have to</s>

Really, just "Provide a valid address."  Is pretty clear in this context IMO